doc/cephadm/operations.rst

```diff
@@ -734,3 +734,62 @@ Purge ceph daemons from all hosts in the cluster
 
   # For each host:
   cephadm rm-cluster --force --zap-osds --fsid <fsid>
+
+
+Replacing a device
+==================
+
+The ``ceph orch device replace`` command automates the process of replacing the underlying device of an OSD.
+Previously, this process required manual intervention at various stages.
```

Comment:
hum.. I think you have to define what a "dead device" is exactly?

Reply:
As in, the device has already failed. At which point the OSDs related to it would already be down. This feature was made with the idea to replace devices that are still functioning on OSDs that are still up iirc.